includes/ResourceLoader/ClientHtml.php

```diff
@@ -496,14 +496,18 @@ public static function makeLoad( Context $mainContext, array $modules, $only,
 						if ( $only === Module::TYPE_STYLES ) {
 							$chunk = Html::linkedStyle( $url );
 						} elseif ( $context->getRaw() ) {
+							/** Fandom change - start **/
+							$config = $mainContext->getResourceLoader()->getConfig();
+							$useDefer = $config->has('ResourceLoaderUseDeferScripts') ? $config->get('ResourceLoaderUseDeferScripts') : false;
```

Comment:
```suggestion
							$useDefer = $config->get( 'ResourceLoaderUseDeferScripts' );
```

Now that I think about it, won't it get cast to a bool anyway if you do it like this?